docs/docs-source/docs/modules/get-started/pages/installing-cloudflow.adoc

```diff
@@ -0,0 +1,361 @@
+----
+
+To apply the `spark-rolebinding.yaml` file execute the following command. Note that this has *only* to be performed if you are installing Cloudflow on Openshift.
+
+  oc apply -f spark-rolebinding.yaml
+
+After the Spark operator installation completes, we need to adjust its webhook configuration.
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS    RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running   0          109s
+spark-operator-sparkoperator-56d6ffc8cd-cln67       1/1     Running   0          109s
+----
+
+For this, we will create yet another YAML file that we will apply to the cluster with `kubectl`.
+----
+cat > spark-mutating-webhook.yaml << 'EOF2'
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cloudflow-patch-spark-mutatingwebhookconfig
+spec:
+  template:
+    spec:
+      serviceAccountName: cloudflow-operator
+      restartPolicy: OnFailure
+      containers:
+        - name: main
+          image: alpine:3.12
+          command:
+            - /bin/ash
+            - "-c"
+            - |
+              apk update && apk add wget
+              wget -q -O /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.16.12/bin/linux/amd64/kubectl && chmod 755 /bin/kubectl
+              NAME="spark-operator-sparkoperator"
+              API_VERSION=$(kubectl get deployment -n cloudflow $NAME -o jsonpath='{.apiVersion}')
+              UUID=$(kubectl get deployment -n cloudflow $NAME -o jsonpath='{.metadata.uid}')
+              KIND=$(kubectl get deployment -n cloudflow $NAME -o jsonpath='{.kind}')
+              HOOK_NAME="spark-operator-sparkoperator-webhook-config"
+              JSON=$(cat <<EOF
+              {
+                "metadata": {
+                  "ownerReferences": [
+                    {
+                      "apiVersion": "$API_VERSION",
+                      "blockOwnerDeletion": true,
+                      "controller": true,
+                      "kind": "$KIND",
+                      "name": "$NAME",
+                      "uid": "$UUID"
+                    }
+                  ]
+                }
+              }
+              EOF
+              )
+              echo $JSON
+              kubectl patch MutatingWebhookConfiguration $HOOK_NAME -n cloudflow -p "$JSON"
+EOF2
+----
+
+Now we can apply the changes to the webhook.
+
+  kubectl apply -f spark-mutating-webhook.yaml --namespace cloudflow
+
+That completes the installation of the Spark operator. 
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS      RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running     0          3m29s
+cloudflow-patch-spark-mutatingwebhookconfig-66xxb   0/1     Completed   0          21s
+spark-operator-sparkoperator-56d6ffc8cd-cln67       1/1     Running     0          3m29s
+----
+
+== Installing the Flink operator
```

Comment:
This section might be worth a separate page, just as Strimzi has its own.

Reply:
Ok